tests/acceptance/features/bootstrap/DataExporterContext.php

```diff
@@ -284,23 +285,42 @@ private function assertFilePhysicallyExistInLastExport($filename, $message = '')
 	 * @return void
 	 */
 	private function assertFilePhysicallyExistInLastExportWithContent($filename, $content, $trash = false) {
-		if ($trash === true) {
+		if ($trash) {
 			$this->featureContext->listTrashbinFileInServerRoot(self::path("$this->lastExportPath/files_trashbin"));
 			$matches = [];
 			foreach (HttpRequestHelper::getResponseXml($this->featureContext->getResponse())->data->element as $item) {
 				$file = (string)$item;
-				if (strpos($file, $filename) === 0) {
+				if (\strpos($file, $filename) === 0) {
 					$matches[] = $file;
 				}
 			}
+			$fileFound = false;
+			$fileCount = \count($matches);
 			foreach ($matches as $match) {
-				$this->featureContext->theFileWithContentShouldExistInTheServerRoot(
-					self::path("$this->lastExportPath/files_trashbin/$match"),
-					$content
+				$path = "$this->lastExportPath/files_trashbin/$match";
+				$this->featureContext->readFileInServerRoot($path);
+				Assert::assertSame(
+					200,
+					$this->featureContext->getResponse()->getStatusCode(),
+					"Failed to read the file {$path}"
 				);
+				$fileContent = HttpRequestHelper::getResponseXml($this->featureContext->getResponse());
+				$fileContent = (string)$fileContent->data->element->contentUrlEncoded;
+				$fileContent = \urldecode($fileContent);
+
+				while ($fileCount > 0) {
+					$fileCount--;
+					if ($fileContent === $content) {
+						$fileFound = true;
+						break 2;
```

Comment:
why `break 2`?
just `break` should work fine..

Reply:
break will terminate current loop while break 2 will terminate 2 nested loops. So just break won't work in this case.